packit/status.py

```diff
@@ -81,32 +84,38 @@ def get_dg_versions(self) -> None:
                 logger.debug(f"Branch {branch} is not present: {ex}")
                 continue
             try:
-                logger.info(f"{branch}: {self.dg.specfile.get_version()}")
+                dg_versions[branch] = self.dg.specfile.get_version()
             except PackitException:
                 logger.debug(f"Can't figure out the version of branch: {branch}")
         self.dg.checkout_branch("master")
 
-    def get_up_releases(self, number_of_releases: int = 5) -> None:
+        return dg_versions
+
+    def get_up_releases(self, number_of_releases: int = 5) -> List:
         """
         Get specific number of latest upstream releases
         :param number_of_releases: int
-        :return: None
+        :return: List
         """
-        latest_releases = self.up.local_project.git_project.get_releases()
-        if len(latest_releases) > 0:
-            logger.info("\nGitHub upstream releases:")
-            # take last five releases
-            latest_releases = (
-                latest_releases[:number_of_releases]
-                if len(latest_releases) > number_of_releases
-                else latest_releases
-            )
-            upstream_releases_str = "\n".join(
-                f"{release.tag_name}" for release in latest_releases
-            )
-            logger.info(upstream_releases_str)
+        latest_releases: List[Release] = []
+        try:
+            latest_releases = self.up.local_project.git_project.get_releases()
+
+            if latest_releases:
+                logger.debug("GitHub upstream releases fetched.")
+                # take last five releases
+                latest_releases = (
+                    latest_releases[:number_of_releases]
```

Comment:
you don't actually need to do this if/else nonsense; `return latest_releases[:number_of_releases]` will work just fine:
```python
In [2]: [1, 2, 3][:5]
Out[2]: [1, 2, 3]

In [3]: [1, 2, 3][:1]
Out[3]: [1]
```